plugin_runner/plugin_runner.py

```diff
@@ -84,7 +84,12 @@ async def HandleEvent(self, request: Event, context: Any) -> EventResponse:
                 compute_start_time = time.time()
                 _effects = await asyncio.get_running_loop().run_in_executor(None, protocol.compute)
                 effects = [
-                    Effect(type=effect.type, payload=effect.payload, plugin_name=base_plugin_name)
+                    Effect(
+                        type=effect.type,
+                        payload=effect.payload,
+                        plugin_name=base_plugin_name,
+                        protocol_classname=protocol.__class__.__name__,
```

Comment:
good call on the naming. i'll change it to just classname. and yes, agree that we should only populate it for CQM classes - it will save us a query lookup in home-app on the receiving side!